app/assets/javascripts/usage_charts.js

```diff
@@ -13,26 +13,21 @@ $(document).ready(function() {
     var chartContainer = $('.usage-chart').first();
     var chartConfig = chartContainer.data('chart-config');
 
-//    var measurement = $('#measurement').val();
     var meter = $("#meter").val();
+    if (meter && meter != 'all') {
+      chartConfig.mpan_mprn = meter;
+    }
 
-    if (meter) {
-      chartConfig.series_breakdown = 'none';
-      if (meter != 'all') {
-        chartConfig.mpan_mprn = meter;
-      } else {
-        chartConfig.mpan_mprn = null;
-      }
-    } else {
-      chartConfig.mpan_mprn = null;
-      chartConfig.series_breakdown = 'meter';
+    var series_breakdown = $("#series_breakdown").val();
+    if (series_breakdown) {
+      chartConfig.series_breakdown = series_breakdown;
     }
 
-//    chartConfig.y_axis_units = measurement;
```

Comment:
Just wanted to check that this isn't needed any more? Looks like there's no axis switching on these charts so possibly not.

Reply:
Yes, as far as I can tell this has been commented out since Feb 2022, and I can't see any field in the usage pages with id=measurement (which is where the value would come from ), so I'm assuming not used any more